packages/escape-html/README.md

```diff
@@ -43,18 +43,7 @@ Returns an escaped attribute value.
 
 _Related_
 
--   <https://w3c.github.io/html/syntax.html#elements-attributes>
-
-"[...] the text cannot contain an ambiguous ampersand [...] must not contain
-any literal U+0022 QUOTATION MARK characters (")"
-
-Note we also escape the greater than symbol, as this is used by wptexturize to
-split HTML strings. This is a WordPress specific fix
-
-Note that if a resolution for Trac#45387 comes to fruition, it is no longer
-necessary for `__unstableEscapeGreaterThan` to be used.
-
-See: <https://core.trac.wordpress.org/ticket/45387>
```

Comment:
In the source, we can change `See: https://core.trac.wordpress.org/ticket/45387` by `@see https://core.trac.wordpress.org/ticket/45387` so the link to track is listed in a separate line.

Any ideas about how to fix the formatting of the comment @saramarcondes ? Not a blocker, but it's definitely not very nice.

Reply:
It's interesting. This new parser has a "preserve" spacing argument but it only seems to respect that _sometimes_. I'll take a look at this, you're right that this doesn't look nice. Thanks for catching it.

Reply:
Found a fix for it, `cleanSpaces` also strips out new lines so we just shouldn't pass it through there.